README.md

```diff
@@ -2,24 +2,84 @@
 
 A dockerized version of [AltServer-Linux](https://github.com/NyaMisty/AltServer-Linux) with Wi-Fi sync capability. 
 
-## Run the server
+## Requirements
 
-### prerequisites
+- Docker
+- Avahi running on host system
+- usbmuxd **NOT** running on host system
 
-The only dependency is docker, but it is important to note two things:
+## Installation
 
-- You might not be able to refresh apps if **usbmuxd** is running in the _host_ system, the docker guest already includes it and binds to the usb interface. Make sure your host system is **NOT** running it.
-- You might need to be running the **avahi-daemon** in the host system.
+### Installing Docker (if not installed already)
```

Comment:
While this is nice to have, I would prefer to just redirect people to the docker website so they can install it themselves, do you mnd removing this? the rest is great, really appreciate your help!

Reply:
Sure! I personally thought the automated install was a neat thing to have, but I can change it to redirect to their install guide.

Reply:
I do also like it but I feel like it's out of the scope of the project in some way